src/ApprenticeAan/SFA.DAS.ApprenticeAan.Application/CalendarEvents/Queries/GetCalendarEvents/CalendarEvent.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace SFA.DAS.ApprenticeAan.Application.CalendarEvents.Queries.GetCalendarEvents;
+public class CalendarEvent
+{
+    public Guid CalendarEventId { get; set; }
+    public string? CalendarName { get; set; }
+    public string? EventFormat { get; set; }
+    public DateTime Start { get; set; }
+    public DateTime End { get; set; }
+
+    public string? Description { get; set; }
+    public string? Summary { get; set; }
+    public string? Location { get; set; }
+    public string? Postcode { get; set; }
+    public double? Longitude { get; set; }
+    public double? Latitude { get; set; }
```

Comment:
Make sure only nullables are set to nullable, to by pass the warning we can use `=null!;`

Reply:
will use that